src/main/Main.hs

```diff
@@ -773,7 +773,7 @@ execCmd ExecOpts {..} go@GlobalOpts{..} =
                 -- Add RTS options to arguments
                 let argsWithRts args = if null eoRtsOptions 
                             then args :: [String]
-                            else args ++ ["+RTS"] ++ eoRtsOptions ++ ["-RTS"]
+                            else args ++ ["+RTS"] ++ concatMap words eoRtsOptions ++ ["-RTS"]
```

Comment:
the --package and --ghc-options flags still use text/stringOption and take multiple arguments I assume these are just that way because of age?

I assume string/textOptions is just legacy method to get a list of these?